### PR TITLE
lesson11

### PR DESCRIPTION
**Pull request by ouvarov**

### CODE REVIEW

File: submissions/Sania/lesson11/js/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const combineArray = (arr) => {
+    return document.getElementsByClassName('combine-array--answer')[0].innerHTML = arr.reduce((newArray, current) => newArray.concat(current), []);
```
Comment (zahar-g) [inline submissions/Sania/lesson11/js/index.js:2]: no needs to `return` anything here

File: submissions/Sania/lesson11/js/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+function every (array, condition) {
+    const answer = document.getElementsByClassName('every--answer')[0];
+    for (const item of array) {
+        if (!condition(item)) {
+            return answer.innerHTML = 'false';
+        }
+    }
+    return answer.innerHTML = 'true';
```
Comment (zahar-g) [inline submissions/Sania/lesson11/js/index.js:12]: no needs to `return` anything here

Comment (ouvarov) [reply submissions/Sania/lesson11/js/index.js:12]: image

**Comment by ouvarov** [comment]
image

**Pull request merged**